bin/nexus-upload-repo-raw

```diff
@@ -36,11 +36,11 @@ upload-asset-raw() {
             -F "raw.directory=$(dirname "$asset")" \
             -F "raw.asset1.filename=$(basename "$asset")" \
             -F "raw.asset1=@${file}")"
-        echo "DEBUG <curl to URL  ${URL}/v1/components?repository=${repo}> completed with: $status"
+        echo "$status"
```

Comment:
IUF will not be able to recognize this message if DEBUG is removed. Extra information is for debugging.